data/governments.txt

```diff
@@ -454,6 +454,29 @@ government "Hai (Wormhole Access)"
 	"hostile hail" "hostile hai"
 	"hostile disabled hail" "hostile disabled hai"
 
+# A government used to have Unfettered raids in Wah Ki and nearby systems.
```

Comment:
```suggestion
# A government that is used to allow Unfettered raids in Wah Ki and nearby systems.
```

Not really incorrect, but "used to have" also implies previously but not longer possessing, so more clarity might prevent future confusion. :)